guides/plugins/plugins/administration/add-shortcuts.md

```diff
@@ -0,0 +1,57 @@
+# Adding Shortcuts
+
+## Overview 
+
+Shortcuts in Shopware 6 are defined on a Component basis. This guide will show you how to add your own ones.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance and full access to both the files and preferably a registered module and custom component.
+Of course you'll have to understand JavaScript, but that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Configuring the Shortcuts
+
+The following code sample will show you how to register shortcuts in your components with help of the `shortcuts` attribute.
+
+{% code title="<plugin root>/src/Resources/app/administration/src/module/swag-example/index.js" %}
+```javascript
+const { Component } = Shopware;
+
+Component.register('swag-basic-example', {
+    
+    shortcuts: {
+        'SYSTEMKEY+S': {
+            active() {
+                return this.acl.can('product.editor');
+            },
+            method: 'myEditProductFunction'
+        },
+        ESCAPE: 'myCancelEditProductFunction'
+    },
+
+   
+    methods: {
+        myEditProductFunction() {
+            console.log("myEditProductFunction")
+        },
+        myCancelEditProductFunction() {
+            console.log("myCancelEditProductFunction")
+        }
+    }
+});
+```
+{% endcode %}
+
+The first keyboard shortcut reacts to the key combination of `SYSTEMKEY+S`, only if the the user has the privilege `product.editor`, with the invocation of the component method with the name `myEditProductFunction`.
```

Comment:
How can I know what `SYSTEMKEY` means? Is there a list like `SYSTEMKEY: Windows Ctrl / Apple Cmd` or something like that? What is the key called for Alt/⌥?

Reply:
I mean. It is not a list but it is the description I looked for.